src/components/NFTPortfolio/SubscriptionForm.vue

```diff
@@ -127,15 +198,22 @@ export default {
         return;
       }
 
+      this.isLoading = true;
+
+      if (!this.isWalletConnected) {
+        await this.handleSubscriptCreator();
+      } else {
+        await this.updateWalletEmail();
+      }
+    },
+    async handleSubscriptCreator() {
       logTrackerEvent(
         this,
         'NFT',
         'nft_mint_subscription_submit',
         this.wallet,
         1
       );
-
-      this.isLoading = true;
```

Comment:
It is best to keep isLoading true/false in the same function for readablity, otherwise we have no idea when and which function set isLoading to true or false